lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -70,6 +73,37 @@ def initialize(serializer: Serialize, **rest)
       # An event was triggered.
       # Re-evaluate all subscribed queries and push the data over ActionCable.
       def execute_all(event, object)
+        execute_legacy(event, object) if config.handle_legacy_subscriptions
+
+        fingerprints = redis.smembers(FINGERPRINTS_PREFIX + event.topic)
+        return if fingerprints.empty?
+
+        grouped_subscription_ids =
+          redis.pipelined do
+            fingerprints.map do |fingerprint|
+              redis.smembers(SUBSCRIPTIONS_PREFIX + fingerprint)
+            end
+          end
+
+        grouped_subscription_ids.each do |subscription_ids|
```

Comment:
Have you considered wrapping this in `redis.pipelined` as well?

Reply:
Also the JSON payload could be computed only once per group

Reply:
Done! Thanks for pointing it out!